src/legal-officer/LegalOfficerPaths.tsx

```diff
@@ -29,11 +29,17 @@ export function transactionsPath(coinId: string): string {
 export const LOC_REQUESTS_RELATIVE_PATH = '/loc';
 export const LOC_REQUESTS_PATH = LEGAL_OFFICER_PATH + LOC_REQUESTS_RELATIVE_PATH;
 
-export const LOC_DETAILS_RELATIVE_PATH = '/loc-details/:locId';
-export const LOC_DETAILS_PATH = LEGAL_OFFICER_PATH + LOC_DETAILS_RELATIVE_PATH;
-export function locDetailsPath(locId: string) {
-    return LOC_DETAILS_PATH.replace(":locId", locId)
+export const TRANSACTION_LOC_DETAILS_RELATIVE_PATH = LOC_REQUESTS_RELATIVE_PATH + '/:locId';
+export const TRANSACTION_LOC_DETAILS_PATH = LEGAL_OFFICER_PATH + TRANSACTION_LOC_DETAILS_RELATIVE_PATH;
+export function transactionLocDetailsPath(locId: string) {
+    return TRANSACTION_LOC_DETAILS_PATH.replace(":locId", locId)
 }
```

Comment:
IMHO rather than dropping the function `locDetailsPath`, it could be used to factor out the logic that appear at several place in the code:
```
export function locDetailsPath(locId: string, locType: string) {
  return locType === 'Transaction' ? transactionLocDetailsPath(request.id) : identityLocDetailsPath(request.id)
}
```
That will ease our life if/when we'll have a new LOC type.

Reply:
Agreed.

Reply:
Will do in another PR.